src/Widgets/FormCheckboxWidget/FormCheckboxWidgetComponent.js

```diff
@@ -26,7 +21,7 @@ Scrivito.provideComponent("FormCheckboxWidget", ({ widget }) => {
         content={widget}
         attribute="label"
         tag="label"
-        {...labelOptions}
+        htmlFor={id}
```

Comment:
With that the checkbox comes selected at least for a fraction of a second:


https://github.com/Scrivito/scrivito_example_app_js/assets/86275/de1e0db8-fe12-4b5e-ad4a-ed07aea310cb

I preferred the previous behavior, where the checkbox does not change on click at all.

Reply:
> FormInputFieldWidgetComponent.js looks good, but FormCheckboxWidgetComponent.js is not ideal

Both behave the same. The difference is only in app styles. Text inputs in this app currently don't have dedicated styles while `:active` (mouse button pressed on the input or label), while checkbox inputs do.

> I preferred the previous behavior, where the checkbox does not change on click at all.

What is the reason to hide the `:active` style in in-place-editing?
One could argue that applying this style in in-place-editing is more WYSIWYG.
For example, an editor would see the difference in styling of text + input vs. label + input without having to switch to preview mode.

Reply:
I see. Then let's keep it like you proposed.